.goreleaser.yaml

```diff
@@ -1,45 +1,68 @@
-# This is an example .goreleaser.yml file with some sensible defaults.
-# Make sure to check the documentation at https://goreleaser.com
-before:
-  hooks:
-    # You may remove this if you don't use go modules.
-    - go mod tidy
-    # you may remove this if you don't need go generate
-    - go generate ./...
+project_name: scratchdata
+
+dist: .builds
+env:
+  - CGO_ENABLED=1
+
 builds:
-  - env:
-      - CGO_ENABLED=1
+  # linux amd64
+  - id: scratchdata-linux-amd64
+    binary: scratchdata
+    main: ./
+    env:
+      - CC=x86_64-linux-gnu-gcc
+      - CXX=x86_64-linux-gnu-g++
     goos:
       - linux
-      - windows
+    goarch:
+      - amd64
+    flags:
+      - -trimpath
+    ldflags:
+      - -extldflags "-lc -lrt -lpthread --static"
```

Comment:
If we compile with these flags, then I get the same error:

```
CGO_ENABLED=1 go build --ldflags="-extldflags=-static" .
```

So I think the `--static` is the main problem here. Do we need any of these flags or ldflags to be set?

Reply:
@poundifdef Not sure, do the project requires any of those. If not we can remove them